src/home_robot_hw/launch/startup_stretch_hector_slam.launch

```diff
@@ -24,6 +24,7 @@
 		<arg name="camera" value="camera"/>
 		<arg name="align_depth" value="true"/>
 		<!--arg name="filters" value="temporal,decimation,spatial,disparity"/-->
```

Comment:
This commented line is the same as the below line?

Reply:
good point. Santhosh had recommended i check something - not seeing an immediate difference w/o decimation. removing the commented out line